keel-sql/src/main/resources/db/changelog/20200302-drop-api-version.yml

```diff
@@ -0,0 +1,17 @@
+databaseChangeLog:
+- changeSet:
+    id: drop-api-version
+    author: fletch
+    changes:
+    - sql:
+        sql: >
+          update resource
+          set kind = concat_ws(
+            '/',
```

Comment:
I thought the point was to switch from slashes to colons because of URIs? 🤔

Reply:
colons in the id. I deliberately tried to make it so the ids don't need to change. The kinds need to contain the context that was previously in api_version

Reply:
Another factor is that I don't want the version from the `api_version` getting into the id. The reason is that id should uniquely identify a resource so that we don't get conflicts. If you have resources with ids `ec2/v1/cluster:prod:keel` and `ec2/v2/cluster:prod:keel` that _is_ a conflict but we wouldn't detect it because `v1` != `v2`.

Reply:
Makes sense. I misunderstood what you were trying to do. Thanks for clarifying.